guide/daily-spending-wallet/requesting/receiving.md

```diff
@@ -0,0 +1,181 @@
+   alt-text = "Screen showing a notification that is asking the user to open their wallet so they can receive their payment."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+With a [lightning wallet server]({{ '/guide/how-it-works/lightning-services/#receive-payments-offline' | relative_url }}), a payment can be intercepted and held for a user until they open their wallet. A push notification can be sent to the user so they can open their wallet and receive the payment. 
+
+Another option is to educate users to keep their wallet open. This can be explained during [first use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }}) or when the user attempts to close their wallet.
+
+</div>
+
+## Receiving lightning payments 
+
+Once the user is online, a payment channel with [inbound liquidity]({{ '/guide/how-it-works/liquidity/'| relative_url }}) is also required to receive the payment. If the user already has a payment channel with enough inbound liquidity the payment will be [received]({{ '/guide/daily-spending-wallet/receiving/#received-payment'| relative_url }}).
+
+If the user does not have a channel open, or one with enough inbound liquidity, a new channel needs to be opened. Any additional fees required to open a channel need to be communicated to users when [creating a payment request]({{ '/guide/daily-spending-wallet/requesting/'| relative_url }}), and before they share it.
+
+Channel opens should be done by a lightning service provider (LSP). Channels should be opened [on-demand]({{ '/guide/how-it-works/lightning-services/#on-demand-liquidity'| relative_url }}) and require [zero confirmations]({{ '/guide/how-it-works/lightning-services/#zero-confirmation'| relative_url }}) so users can receive and spend their bitcoin instantly.
+
+## Receiving on-chain payments
+
+To receive on-chain payments and maintain a single lightning balance a [lightning wallet server (LWS)]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers'| relative_url }}) is required to conduct [submarine swaps](https://blog.muun.com/a-closer-look-at-submarine-swaps-in-the-lightning-network/) on the users behalf. Submarine swaps move bitcoin between the bitcoin and lightning networks.
+
+{% include tip/open.html color="blue" icon="info" label="Experimental options" %}
+
+Experimental options such as [Peerswap](https://www.peerswap.dev/) and [Collaborative funding](https://bitcoinops.org/en/topics/dual-funding/) may be more trustless options that could be used to receive on-chain payments whilst maintaining a single lightning balance.
+
+{% include tip/close.html %}
+
+### Refunds
+
+There may be scenarios where a submarine swap is not completed successfully, such as when the wallets LSP is offline. In these cases, the users will need to claim the on-chain funds that failed to be swapped by sending them to a on-chain address. 
```

Comment:
> users will need to claim the on-chain funds that failed to be swapped by sending them to a on-chain address. 

Can you elaborate a bit more on this part? I am not sure it's clear enough.

Reply:
I assume the LSP will try this multiple times before going this refund route?

Reply:
Will clarify this, yes they can. The LWS is actually who does the swap* Need to fix that. The LSP opens a channel though if it is required. 

Some context on how Breez deals with these refunds: https://doc.breez.technology/Adding-Funds-via-Submarine-Swaps.html?highlight=refund#how-can-users-claim-their-refunds